cves/kernel/CVE-2020-12652.yml

```diff
@@ -141,10 +132,13 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: |
+    There are some tests around, I don't believe they apply to the race
+    condition that occurs.
+  fix: false
+  fix_answer: |
+    Not automated test added related to this vulnerability.
```

Comment:
Should be "no" instead of "not" just to make it grammatically correct.

Reply:
This comment didn't show up in the file previews for some reason, but it is addressed now.